common/djangoapps/course_modes/views.py

```diff
@@ -190,11 +192,22 @@ def get(self, request, course_id, error=None):
                 for x in verified_mode.suggested_prices.split(",")
                 if x.strip()
             ]
+            min_price = verified_mode.min_price
+
             context["currency"] = verified_mode.currency.upper()
-            context["min_price"] = verified_mode.min_price
+            context["min_price"] = min_price
```

Comment:
Is there a reason to set verified_mode.min_price as its own variable?

Reply:
i'm using it below a couple times (i actually noticed i used verified_mode.min_price below a couple times, i'll fix that)

Reply:
nit-picky, but can you name it something that shows that it's the old min_price, before you update it?